plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaPageSink.java

```diff
@@ -52,14 +52,13 @@ public KafkaPageSink(
             List<KafkaColumnHandle> columns,
             RowEncoder keyEncoder,
             RowEncoder messageEncoder,
-            KafkaProducerFactory producerFactory)
+            KafkaProducer<byte[], byte[]> producer)
```

Comment:
undo, if you pass producer as argument here then ownership of this object is not obvious.